frontend/src/components/story/StoryPeriod.vue

```diff
@@ -22,5 +18,27 @@ export default {
     period: { type: Object, required: true },
     editing: { type: Boolean, default: false },
   },
+  data() {
+    return {
+      expandedDays: [],
+    }
+  },
+  watch: {
+    period: {
+      immediate: true,
+      handler(value) {
+        this.computeExpandedDays(value)
+      },
+    },
+  },
+  methods: {
+    computeExpandedDays(period) {
+      period.days()._meta.load.then((days) => {
+        this.expandedDays = days.items.map((day, idx) =>
+          Date.parse(day.end) >= new Date() ? idx : null
```

Comment:
Possible improvement: Once the period has finished completely, the days could all be expanded again

Reply:
Good point => [f7edaf4](https://github.com/ecamp/ecamp3/pull/3038/commits/f7edaf4131cbd0ed1ba471f198a26da7ea36b681)